### DISCUSSION TITLE
DITA arena on the tekom/tcworld conference in Stuttgart/Germany (Nov. 2023)

### DISCUSSION THREAD

**Discussion by stefan-jung** [dita-ot/dita-ot#4166] [Ideas]
Hi all,

because the Convex/DITA Europe probably will not take place in Europe anymore, I'm trying to get more attention for DITA on the tekom/tcworld conference. I will be the program manager of a "DITA arena" this year. Click [here](https://www.youtube.com/watch?v=3wfYBk6UXRw) to get some impressions of the conference last year and click [here](https://www.technical-communication.org/translate-to-englisch-tcworld-conference-2022-a-review) to see a picture of an arena. The call-for-paper works a little bit different for the arenas. This means, you can just write to me and do not have to go through the official conference portal.

* Short (15 min) and longer (45 min) sessions are possible.
* Arena speakers get a ticket for the **fair** for all days and a ticket for the **conference** (the expensive one) for the arena day.
* I need to present the arena program in September. 

Do you like the idea? 
Would you like to present? 
Do you have more ideas for additional wokring sessions? 
How should the environment look like to make you feel comfortable?

Please share your thoughts. Hope to see you in person in November in Stuttgart.

**Comment by raducoravu** [comment]
Sounds interesting @xephon2, is there a particular way to register a proposed presentation? Or should we register it through the regular Tekom conference tool?

**Reply by stefan-jung** [reply]
Please send me a mail, you cannot apply for the DITA Arena through the tekom portal.
